kensu/boto3/__init__.py

```diff
@@ -81,19 +83,146 @@ def kensu_put(event_params, event_ctx, **kwargs):
             kensu.add_dependencies_mapping(short_result_sc.to_guid(),str(col),input_schema_pk,str(col),'s3_put')
         kensu.report_with_mapping()
 
+
+def fmt_timestream_uri(db, table):
+    return f"aws::TimeStream://{db}.{table}"
+
+def fmt_timestream_name(db, table):
+    return f"TimeStream://{db}.{table}"
+
+
+def fmt_timesteam_lds_names(db, table):
+    return ['logical::' + fmt_timestream_name(db, table),
+            'logicalLocation::' + fmt_timestream_uri(db, table),
+            ]
+
+def kensu_timestream_query(event_params):
+    logging.warning('KENSU: handler of timestream_query.query...')
+    # if true, we'll report all columns found in the table, doesn't matter what columns were actually referred
+    # FIXME: do we care about exact columns which were referred?
+    ALWAYS_REPORT_ALL_COLUMNS = True
```

Comment:
this one controls if we take send full columns (as in `describe db.table`) or explicit columns (only the ones found in SQL query, but this one might miss some columns)